tools/c7n_tencentcloud/c7n_tencentcloud/resources/cbs_snapshot.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod, isoformat_date_str
+
+
+@resources.register("cbs-snapshot")
+class CBSSnapshot(QueryResourceManager):
+    """cbs-snapshot"""
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "SnapshotId"
+        endpoint = "cbs.tencentcloudapi.com"
+        service = "cvm"
+        version = "2017-03-12"
+        enum_spec = ("DescribeSnapshots", "Response.SnapshotSet[]", {})
+        metrics_instance_id_name = "SnapshotId"
+        paging_def = {"method": PageMethod.Offset, "limit": {"key": "Limit", "value": 20}}
+        resource_prefix = "volume"
+        taggable = True
+
+    def augment(self, resources):
+        for resource in resources:
+            isoformat_date_str(resource, ["CreateTime"], "%Y-%m-%d %H:%M:%S")
```

Comment:
what's the server side response time tz?

Reply:
it looks san tz in the format, is there an implied tz ? "CreatedTime\": \"2022-09-29 23:28:28\"

Reply:
the response time format is complicated, some are +0800, some are utc, what is worse, most of them has no tz info.